pkg/capr/planner/drain.go

```diff
@@ -12,12 +12,15 @@ import (
 	"github.com/rancher/wrangler/v3/pkg/kv"
 )
 
-func getRestartStamp(plan *plan.NodePlan) string {
+func getDrainHash(plan *plan.NodePlan) string {
 	for _, instr := range plan.Instructions {
+		if instr.Name != "install" {
```

Comment:
does this mean we won't drain when doing etcd snapshot restore or encryption key rotation?